app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -246,12 +246,12 @@ void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex,
         ProgramPtr program = create->GetProgramFactory()->CreateProgram(create, 0);
         RenderStatePtr state = RenderState::Create(create);
         state->SetProgram(program);
-        vrb::Color handColor = vrb::Color(0.5f, 0.5f, 0.5f);
+        vrb::Color handColor = vrb::Color(0.85f, 0.85f, 0.85f);
         handColor.SetAlpha(1.0);
-        state->SetMaterial(handColor, handColor,
-                           Color(0.0f, 0.0f, 0.0f),
-                           0.0f);
-        state->SetLightsEnabled(false);
+        state->SetMaterial(handColor, Color(0.5f, 0.5f, 0.5f),
+                           Color(0.5f, 0.5f, 0.5f),
+                           0.75f);
+        state->SetLightsEnabled(true);
```

Comment:
@felipeerias is reporting that this is causing weird reflections in the spheres, maybe we don't need them.

Reply:
He submitted a fix for this in https://github.com/Igalia/wolvic/pull/677 and it is now integrated. The spheres look great now with the lighting.

It is true that this change is not relevant for the hand models, so I can drop it if you want. But IMHO it does improve the look and feel of the joint spheres on Pico.

Reply:
OK, let's keep it then